project.clj

```diff
@@ -63,7 +63,6 @@
   :java-source-paths ["src/com"]
   :aliases {"code-coverage" ["with-profile" "test" "cloverage" "--output" "coverage" "--lcov"]
             "test-cluster"  ["shell" "lein" "test"]}
```

Comment:
Can we explore an alternative to this: `"test-cluster"  ["shell" "lein" "test"]}`

It seems convoluted - use makefile from shell to invoke lein and then use lein to invoke shell

There should be a simpler way of doing this.

Reply:
I agree, even me I got confused at first. Looking into ways to have a simpler/cleaner implementation

Reply:
done